src/IssueViz.UnitTests/Editor/LocationTagging/LocationTagger_BufferChangeTests.cs

```diff
@@ -56,7 +56,7 @@ public void BufferChanged_FileIsRenamed_CurrentFileNameIsUsed()
             RaiseBufferChangedEvent(bufferMock, bufferMock.Object.CurrentSnapshot, bufferMock.Object.CurrentSnapshot);
 
             // Check the store was notified using the new name
-            storeMock.Verify(x => x.Refresh(new[] { changedName }), Times.Once);
+            storeMock.Verify(x => x.RefreshOnBufferChanged(changedName), Times.Once);
```

Comment:
Same change everywhere - change call from `Refresh(files)` to `RefreshOnBufferChanged(file)`